### ISSUE TITLE
Helm function fail after upgrade from 2.8.2 to 2.9.0

### ISSUE THREAD

**Issue by zhangtbj** [kubernetes/helm#4021]
Output of `helm version`: 2.9.0

Output of `kubectl version`:1.10

I upgraded my helm version from 2.8.2 to 2.9.0, then I also upgraded the tiller version to 2.9.0, after upgrade.  I cannot use my helm correctly, it report:
```
✘-1 $ helm list
Error: Get http://localhost:8080/api/v1/namespaces/kube-system/configmaps?labelSelector=OWNER%!D(MISSING)TILLER: dial tcp 127.0.0.1:8080: connect: connection refused

```

I executed on two kube clusters and linux and mac, I also see this error.

My kube cluster is on the IBM Cloud Service, it is not in localhost, I am not sure if helm hardcode `localhsot` in somewhere that only find local kube api.

**Comment by bacongobbler** [comment]
Closing as a duplicate of #4020 which is also a duplicate of #3985 :)

**Comment by zhangtbj** [comment]
OK Thanks! @bacongobbler , BTW when will you have the new version to fix this issue?

Right now I install the helm by this way:
```
# Install helm
RUN mkdir /gopath/helm \
&& curl -L https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get -o /gopath/helm/get_helm.sh \
&& chmod +x /gopath/helm/get_helm.sh \
&& /gopath/helm/get_helm.sh \
&& helm init --client-only
```

SO I think this is the official version, and we have to use the latest version. Pls help provide the new workable version, soon. Thanks!

**Comment by bacongobbler** [comment]
This has been fixed in helm v2.9.1, which is available here: https://github.com/kubernetes/helm/releases/v2.9.1

Thanks everyone!
